src/storage.rs

```diff
@@ -25,40 +43,44 @@ impl From<KeyPrefixU8> for KeyPrefix {
             0x3 => Self::Code,
             0x4 => Self::Storage,
             0x5 => Self::RelayerEvmAddressMap,
+            // 0x6 used
             _ => unreachable!(),
         }
     }
 }
 
 #[allow(dead_code)]
 pub fn bytes_to_key(prefix: KeyPrefix, bytes: &[u8]) -> Vec<u8> {
-    [&[prefix as u8], bytes].concat()
+    [&[VersionPrefix::V1 as u8], &[prefix as u8], bytes].concat()
 }
 
 #[allow(dead_code)]
 pub fn address_to_key(prefix: KeyPrefix, address: &Address) -> [u8; 21] {
     let mut result = [0u8; 21];
-    result[0] = prefix as u8;
-    result[1..].copy_from_slice(&address.0);
+    result[0] = VersionPrefix::V1 as u8;
+    result[1] = prefix as u8;
+    result[2..].copy_from_slice(&address.0);
     result
 }
```

Comment:
I think we need to have the result 22 bytes long instead (1 byte for version + 1 byte for prefix + 20 bytes for address)

Reply:
This is why we should always `[2..21]` as I had warned before. We can turn the Clippy lint on for this post-launch.